app/routers/router.ts

```diff
@@ -200,8 +200,8 @@ export default function router({ skipEarthdataLogin = 'false' }: RouterConfig):
   result.get('/versions', asyncHandler(getVersions));
   result.use('/docs/api', swaggerUi.serve, swaggerUi.setup(yaml.load(ogcCoverageApi.openApiContent), { customJs: '/js/docs/analytics-tag.js' }));
   result.get(collectionPrefix('wms'), asyncHandler(service(serviceInvoker)));
-  result.get(/^.*?\/ogc-api-coverages\/.*?\/collections\/.*?\/coverage\/rangeset/, asyncHandler(service(serviceInvoker)));
-  result.post(/^.*?\/ogc-api-coverages\/.*?\/collections\/.*?\/coverage\/rangeset/, asyncHandler(service(serviceInvoker)));
+  result.get(/^.*?\/ogc-api-coverages\/.*?\/collections\/.*?\/coverage\/rangeset$/, asyncHandler(service(serviceInvoker)));
```

Comment:
Note that there is one case that this returns a 404 now, but used to succeed before (when the url included a trailing slash on the endpoint).

We could keep the same behavior as before by allowing `/rangeset\/?$/`, but it probably should have always failed. I think the change in the behavior probably makes sense to not allow a trailing slash, but wanted to point it out in case there is a reason we should allow it.

Reply:
It makes sense to support the / at the end. I will update the regex.